src/command_modules/azure-cli-sqlvirtualmachine/azure/cli/command_modules/sqlvm/__init__.py

```diff
@@ -0,0 +1,30 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@yareyes @tjprescott, shouldn't this be....

`src/command_modules/azure-cli-sqlvm/azure/cli/command_modules/sqlvm/__init__.py
`
or

`src/command_modules/azure-cli-sqlvirtualmachine/azure/cli/command_modules/sqlvirtualmachine/__init__.py`
instead of:

`src/command_modules/azure-cli-sqlvirtualmachine/azure/cli/command_modules/sqlvm/__init__.py`

import statements use azure.cli.command_modules.sqlvm. I believe the other modules have consistent naming.

Reply:
Good point. Since this PR is merged, you should probably open an issue for this to be fixed..

Reply:
I will submit a PR to fix this!